src/utils.js

```diff
@@ -345,7 +345,7 @@ function getTrustNetworkStatus(
     default:
       query = {
         query: `{
-          trusts(where: { userAddress: "${safeAddress}" }) {
+          trusts(where: { userAddress: "${safeAddress}", limitPercentage_not: ${NO_LIMIT_PERCENTAGE} }) {
```

Comment:
why addint this `limitPercentage_not`?

Reply:
I added in the query because the `limitPercentage` not being the `NO_LIMIT_PERCENTAGE` was being checked in the code already. So, if we can query the data filtering the limit percentage, why check it with coding?